Profiler/TorchUtils/DisposableUtils.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TorchUtils
+{
+    internal static class DisposableUtils
+    {
+        public static T AddedTo<T>(this T self, ICollection<IDisposable> bag) where T : IDisposable
```

Comment:
Don't really like putting stuff that has a very specific use case as an extension on such a popular type.  Can you limit T to a more specific type?

Reply:
Yup these things carried out of my personal utils symlink. Deleted in the next fix